### ISSUE TITLE
De-holiday the PNI

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4012]
Things to do after Christmas break:

Design anytime:
- [x] Update homepage banner
- [x] Update OG graphic 

Dev at the start of Jan:
- [ ] Update homepage banner
- [ ] Update OG graphic 

@caltrider can you think of anything else?

**Comment by alanmoo** [comment]
Is this estimate just for Design?

**Comment by kristinashu** [comment]
Yeah, just design.

**Comment by natalieworth** [comment]
Different color/ treatment options are in the Redpen [here](https://redpen.io/p/dz7a1546b00fdbb6e4). I still feel like the white/ plain one is the best for the homepage since there is a lot going on as user scrolls. For social, we could use a colored option so it stands out more.

**Comment by kristinashu** [comment]
Nat, could you please mockup a couple options for social so we know what you mean?

**Comment by natalieworth** [comment]
Social image and new evergreen header for 2020 [are in folder here](https://drive.google.com/drive/folders/1RGBHgkb923dZo0rEs43EKs-SAT68Mc-r?usp=sharing).

**Comment by natalieworth** [comment]
@kristinashu Should I open a new ticket for dev?

**Comment by alanmoo** [comment]
Yes, please, for the purposes of estimates and velocity tracking.

**Comment by kristinashu** [comment]
Dev ticket is here https://github.com/mozilla/foundation.mozilla.org/issues/4145. @natalieworth just needs to make a small update today and then the dev ticket will be unblocked.
